packages/react-router/src/preloader.tsx

```diff
@@ -47,6 +50,24 @@ export function createApolloLoaderHandler(
   };
 }
 
+
+export const apolloContext = unstable_createContext<ApolloContext>();
+
+export function createApolloMiddleware(
```

Comment:
Same as `createApolloLoaderHandler`, but sets the context value instead of making the user wrap their loader with another function.